### ISSUE TITLE
Cliqz "Tour" and "Offboarding" separate pages are displayed only for first install / uninstall

### ISSUE THREAD

**Issue by SoftVision-PaulOiegas** [mozilla/testpilot#1979]
[Affected versions]:
- Firefox 50.1.0 and up

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Prerequisites]:
- German localized version of Firefox was installed / extracted.
- The "testpilot.env" string pref was manually added with "dev" value on a new clean profile in "about:config" page.
- Test Pilot add-on was previously installed on the profile.

[Steps to reproduce]:
1. Open DE localized Firefox browser with the profile from prerequisites and navigate to "https://testpilot.dev.mozaws.net" page.
2. Click on "Cliqz" experiment card.
3. While observing browser behavior, choose to Enable and Disable Cliqz for multiple times.

[Expected result]:
- The "Tour" and "Offboarding" pages are displayed after each enable / disable.

[Actual result]:
- Cliqz "Tour" and "Offboarding" pages are displayed only for first install / uninstall

[Notes]:
- This behavior does not change even if browser cache is cleared or Test Pilot is reinstalled.

**Comment by ghost** [comment]
This is referring to the cliqz onboarding pages.  /cc @past

**Comment by past** [comment]
In the future please file issues affecting Cliqz in their [tracker](https://github.com/cliqz-oss/browser-core/issues). This specific behavior seems fine to me in order to inform but not annoy the user.

**Comment by luciancor** [comment]
Thanks guys for the report!

Cliqz "Tour" and "Offboarding" work as designed. We chose to show onboarding only once at the very beginning and off boarding only once per uninstalled version. Please check here - https://github.com/cliqz-oss/browser-core/blob/master/specific/firefox/cliqz@cliqz.com/modules/Extension.jsm#L352

Is there any requirement regarding when these pages should be shown?

**Comment by SoftVision-PaulOiegas** [comment]
I don't think we have any specific requirement. I've logged this because on Test Pilot everything is erased after uninstall in order to leave no trace of the installation. This makes our own onboarding page to be shown each time when TP is installed since we have nothing to tell us that the add-on was previously installed. 
So I assumed that maybe we want the same thing for Cliqz installation behavior trough Test Pilot. But judging on the above comments, it seams not. Closing this.
